### PR TITLE
get rid of ember-cli-qunit dependency

### PR DESCRIPTION
**Pull request by choheekim**
This pr contains three changes:
1. Removing dependency in ember-cli-qunit - ember-cli-qunit is getting deprecated and Ember Exam's using ember-cli-qunit to invoke ember-qunit as a wrapper.
2. Disable sandbox when running Chrome inside a container in testem.js
3. Pass { setupEmberOnerrorValidation: false } param to disable running Ember.onerror validation test in ember-exam

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 language: node_js
 node_js:
-  - "4"
```
Comment (rwjblue) [inline .travis.yml]: This seems somewhat unrelated, and generally considered breaking (since we would no longer be testing Node 4).

Could you make this change in a separate PR and update the `package.json`'s `engines` field? That will help keep things together for the changelog...

Comment (choheekim) [reply .travis.yml]: Open a pr with the change:
https://github.com/ember-cli/ember-exam/pull/148

**Comment by rwjblue** [comment]
OK, should be ready with a rebase...
